snippets/examples-cards.mdx

```diff
@@ -0,0 +1,50 @@
+import { Example } from './examples'
+
+export const ExamplesCards = ({
+  examples,
+  runtime,
+}) => {
+  const examplesData = {
```

Comment:
Sorry, Mintlify wouldn't let me use TypeScript or let me load the data in from a separate doc. I had to put everything inside the component.